examples/exception_handling/exception_handling.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+from typing import Callable, Tuple
+
+from golem_core import GolemNode, Payload, commands
+from golem_core.default_logger import DefaultLogger
+from golem_core.default_payment_manager import DefaultPaymentManager
+from golem_core.low.activity import Activity
+from golem_core.low.exceptions import BatchError
+from golem_core.mid import (
+    Buffer,
+    Chain,
+    Limit,
+    Map,
+    default_create_activity,
+    default_create_agreement,
+    default_negotiate,
+)
+
+PAYLOAD = Payload.from_image_hash(
+    "9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae"
+)
+SUBNET = "public"
+
+
+async def trigger_exception(activity: Activity) -> str:
```

Comment:
I'm not sure what is the exact purpose of the example.
BUT if you want a "readable way of creating an exception in RUN command" i'd try something like

```
async def run_invalid_command(activity: Activity) -> None:
    batch = await activity.execute_commands(
        commands.Deploy(),
        commands.Start(),
        commands.Run("/ooops/no_such_command"),
    )
    await batch.wait(timeout=20)
````

But I might be missing the point of the example.

Reply:
Good point, added